experiment/coverage_utils.py

```diff
@@ -292,3 +406,170 @@ def extract_covered_regions_from_summary_json(summary_json_file):
+
+        # Left-Join to collect function IDs in benchmark df.
+        df_container.segment_df = pd.merge(df_container.segment_df,
+                                           df_container.benchmark_name_df,
+                                           on='benchmark',
+                                           how='outer')
+        # Drop function_name column as we have benchmark IDs now.
+        df_container.segment_df = df_container.\
+            segment_df.drop(columns=['benchmark'])
+
+        # Left-Join to collect function IDs in function df.
+        df_container.function_df = pd.merge(df_container.function_df,
+                                            df_container.function_name_df,
+                                            on='function_name',
+                                            how='outer')
+        # Drop function_name column as we have function IDs now.
+        df_container.function_df = df_container.\
+            function_df.drop(columns=['function_name'])
+
+        # Left-Join to collect function IDs in fuzzer df.
+        df_container.function_df = pd.merge(df_container.function_df,
+                                            df_container.fuzzer_name_df,
+                                            on='fuzzer',
+                                            how='outer')
+        # Drop function_name column as we have fuzzer IDs now.
+        df_container.function_df = df_container.\
+            function_df.drop(columns=['fuzzer'])
+
+        # Left-Join to collect function IDs in benchmark df.
+        df_container.function_df = pd.merge(df_container.function_df,
+                                            df_container.benchmark_name_df,
+                                            on='benchmark',
+                                            how='outer')
+        # Drop function_name column as we have benchmark IDs now.
+        df_container.function_df = df_container.\
+            function_df.drop(columns=['benchmark'])
+
+    except Exception:  # pylint: disable=broad-except
+        logger.error('Error occurred when wrangling for CSV generation.')
+
+
+def generate_segment_and_function_csv_files(df_container):
+    """generates segment, function, function_name, file_name, fuzzer
+    and benchmark csv from DataFrames. All file together contain
+    experiment specific  information"""
+    try:
+        # store merged function csv in local fs or gutils.
+        combined_function_csv_src = os.path.join(get_coverage_info_dir(),
+                                                 'data', 'functions.csv')
+        combined_function_csv_dst = exp_path.filestore(
+            combined_function_csv_src)
+        df_container.function_df.to_csv(combined_function_csv_src, index=False)
+        filestore_utils.cp(combined_function_csv_src,
+                           combined_function_csv_dst,
+                           expect_zero=False)
+
+        # store merged segment csv in local fs or gutils.
```

Comment:
Capitalize store.
Also local vs gsutil is not a distinction that is relevant here, just call it filestore.